docs/topics/multiplatform/multiplatform-run-tests.md

```diff
@@ -1,67 +1,375 @@
-or test framework.
+In this tutorial, you'll learn how to create, configure, and run tests in Kotlin Multiplatform applications.
 
-## Required dependencies
+> This tutorial assumes that you are familiar with:
+> * The layout of a Kotlin Multiplatform project. If this is not the case,
+    complete [this tutorial](multiplatform-mobile-getting-started.md) to get started.
+> * The basics of popular unit testing frameworks, such as [JUnit](https://junit.org/junit5/).
+>
+{type="tip"}
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for multiplatform tests. When you
-[create a multiplatform project](multiplatform-library.md), the Project Wizard automatically adds test dependencies to
-common and platform-specific source sets. 
+Tests for multiplatform projects can be divided into two categories:
 
-If you didn't use the Project Wizard to create your project, you can [add the dependencies manually](gradle-configure-project.md#set-dependencies-on-test-libraries).
+* Tests for common code. These tests can be run on any platform using any supported framework.
+* Tests for platform-specific code. These are essential to test platform-specific logic. They use a platform-specific
+  framework and can benefit from its additional features, such as a richer API and a wider range of assertions.
 
-## Run tests for one or more targets
+Both categories are supported in multiplatform projects. This tutorial will show you how to create, set up, and run unit
+tests for both common and platform-specific code.
 
-To run tests for all targets, run the `check` task.
+## Create a sample project
 
-To run tests for a particular target suitable for testing, run a test task `<targetName>Test`.
+1. Check your environment for multiplatform
+   development. [Install all the necessary tools and update them to the latest versions](multiplatform-mobile-setup.md).
+2. In Android Studio, select **File | New | New Project**.
+3. Select **Kotlin Multiplatform App** in the list of project templates, and click **Next**.
 
-## Test shared code
+   ![Mobile Multiplatform project template](multiplatform-mobile-project-wizard-1.png){width=700}
 
-For testing shared code, you can use [actual declarations](multiplatform-connect-to-apis.md) in your tests.
+4. Specify a name for your application, and click **Next**.
 
-For example, to test the shared code in `commonMain`:
+   ![Mobile Multiplatform project - general settings](multiplatform-mobile-project-wizard-2.png){width=700}
 
-```kotlin
-expect object Platform {
-    val name: String
-}
+5. Leave the **Add sample tests to Shared Module** option unchecked.
 
-fun hello(): String = "Hello from ${Platform.name}"
+   This option adds extra source sets and sample code to assist you with code testing. However, to understand how to
+   create and configure tests better, you'll add them manually in this tutorial.
```

Comment:
That's basically a description that won't be used in the project. We don't ask to take any actions here. It could be a tip, but why do you think it should be a warning?

Reply:
We can put it under the image though

Reply:
It was to make sure that the reader understood to leave the option unchecked but after reviewing it a second time, I don't think it's necessary.